### ISSUE TITLE
Collaboration with other forks?

### ISSUE THREAD

**Issue by norrisjeremy** [mwiede/jsch#32]
There appears to be at least two other JSch forks on Github:
 - [is/jsch](https://github.com/is/jsch)
 - [sschlib/sschlib](https://github.com/sschlib/sschlib)

I wonder if it would be worthwhile to attempt to contact those project owners and investigate the possibility of collaborating on a single joint fork?

**Comment by norrisjeremy** [comment]
FYI, it may also be worthwhile to investigate the issue backlog at the [is/jsch](https://github.com/is/jsch) fork.
It appears that several users have opened issues over the years on this particular fork.

**Comment by mwiede** [comment]
yes I have also researched about other forks and I was trying https://techgaun.github.io/active-forks/index.html to get activity stats

is/jsch does not seems active and does not release its code. But maybe it makes sense to comment on its issues to make them open them here again.

**Comment by manishbansal8843** [comment]
My company is also a heavy user of this library. We can say its their bread and butter. As of now we are using 0.1.54 with added fixes. Let me know if I can help anyway.

**Comment by norrisjeremy** [comment]
Hi @manishbansal8843,

Do you have any changes/bugfixes that your company would be willing to contribute to this fork?

Thanks,
Jeremy
